# New Analysess Kraken2

# P02

/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 20 --gzip-compressed contigs.fasta.gz --output SP2_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP2_BacteriaNCBI_K2_Metagenome.txt
/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP2_BacteriaNCBI_K2_Metagenome.txt -o Teste.bracken -r 151 -l F -t 10

# P03
# 1672723 sequences classified (25.28%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP03-P03/Metagenome3/contigs.fasta.gz --output SP03_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP03_BacteriaNCBI_K2_Metagenome.txt

# P04
# 1220685 sequences classified (24.69%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP04-P04/Metagenome4/contigs.fasta.gz --output SP04_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP04_BacteriaNCBI_K2_Metagenome.txt

# P05
# 1242066 sequences classified (22.67%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP05-P05/Metagenome5/contigs.fasta.gz --output SP05_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP05_BacteriaNCBI_K2_Metagenome.txt

# P06
# 1759240 sequences classified (29.92%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP06-P06/Metagenome6/contigs.fasta.gz --output SP06_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP06_BacteriaNCBI_K2_Metagenome.txt

# P07
# 1675953 sequences classified (25.19%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP07-P07/Metagenome7/contigs.fasta.gz --output SP07_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP07_BacteriaNCBI_K2_Metagenome.txt

# P08
# 1189312 sequences classified (21.74%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP08-P08/Metagenome8/contigs.fasta.gz --output SP08_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP08_BacteriaNCBI_K2_Metagenome.txt

# P09
# 915760 sequences classified (20.46%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP09-P09/Metagenome9/contigs.fasta.gz --output SP09_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP09_BacteriaNCBI_K2_Metagenome.txt

# P10
# 1298434 sequences classified (23.20%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP10-P10/Metagenome10/contigs.fasta.gz --output SP10_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP10_BacteriaNCBI_K2_Metagenome.txt

# P12
# 1467042 sequences classified (24.00%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP12-P12/Metagenome12/contigs.fasta.gz --output SP12_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP12_BacteriaNCBI_K2_Metagenome.txt

# P13
# 1145772 sequences classified (22.79%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP13-P13/Metagenome13/contigs.fasta.gz --output SP13_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP13_BacteriaNCBI_K2_Metagenome.txt

# P14
# 1104833 sequences classified (21.28%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP14-P14/Metagenome14/contigs.fasta.gz --output SP14_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP14_BacteriaNCBI_K2_Metagenome.txt

# P15
# 1518886 sequences classified (25.58%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP15-P15/Metagenome15/contigs.fasta.gz --output SP15_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP15_BacteriaNCBI_K2_Metagenome.txt

# P16
# 1468445 sequences classified (23.65%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP16-P16/Metagenome16/contigs.fasta.gz --output SP16_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP16_BacteriaNCBI_K2_Metagenome.txt

# P17
# 1078693 sequences classified (22.97%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP17-P17/Metagenome17/contigs.fasta.gz --output SP17_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP17_BacteriaNCBI_K2_Metagenome.txt

# P18
# 1465632 sequences classified (26.77%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP18-P18/Metagenome18/contigs.fasta.gz --output SP18_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP18_BacteriaNCBI_K2_Metagenome.txt

# P19
# 1537098 sequences classified (24.02%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP19-P19/Metagenome19/contigs.fasta.gz --output SP19_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP19_BacteriaNCBI_K2_Metagenome.txt

# P20
# 1226031 sequences classified (20.96%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP20-P20/Metagenome20/contigs.fasta.gz --output SP20_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP20_BacteriaNCBI_K2_Metagenome.txt

# P21
# 1487608 sequences classified (23.43%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP21-P21/Metagenome21/contigs.fasta.gz --output SP21_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP21_BacteriaNCBI_K2_Metagenome.txt

# P22
# 1342445 sequences classified (22.89%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP22-P22/Metagenome22/contigs.fasta.gz --output SP22_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP22_BacteriaNCBI_K2_Metagenome.txt

# P23
# 1188865 sequences classified (21.21%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP23-P23/Metagenome23/contigs.fasta.gz --output SP23_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP23_BacteriaNCBI_K2_Metagenome.txt

# P24
# 1100041 sequences classified (22.89%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP24-P24/Metagenome24/contigs.fasta.gz --output SP24_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP24_BacteriaNCBI_K2_Metagenome.txt

# P25
# 1253476 sequences classified (24.23%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP25-P25/Metagenome25/contigs.fasta.gz --output SP25_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP25_BacteriaNCBI_K2_Metagenome.txt

# P26
# 1000623 sequences classified (20.47%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP26-P26/Metagenome26/contigs.fasta.gz --output SP26_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP26_BacteriaNCBI_K2_Metagenome.txt

# P27
# 1446115 sequences classified (25.42%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP27-P27/Metagenome27/contigs.fasta.gz --output SP27_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP27_BacteriaNCBI_K2_Metagenome.txt

# P28
# 1440967 sequences classified (23.33%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP28-P28/Metagenome28/contigs.fasta.gz --output SP28_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP28_BacteriaNCBI_K2_Metagenome.txt

# P29
# 1427154 sequences classified (23.98%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP29-P29/Metagenome29/contigs.fasta.gz --output SP29_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP29_BacteriaNCBI_K2_Metagenome.txt

# P30
# 1354686 sequences classified (22.30%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP30-P30/Metagenome30/contigs.fasta.gz --output SP30_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP30_BacteriaNCBI_K2_Metagenome.txt

# P31
# 1903696 sequences classified (26.01%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP31-P31/Metagenome31/contigs.fasta.gz --output SP31_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP31_BacteriaNCBI_K2_Metagenome.txt

# P32
# 1642838 sequences classified (24.69%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP32-P32/Metagenome32/contigs.fasta.gz --output SP32_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP32_BacteriaNCBI_K2_Metagenome.txt

# P33
# 1490071 sequences classified (28.38%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP33-P33/Metagenome33/contigs.fasta.gz --output SP33_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP33_BacteriaNCBI_K2_Metagenome.txt

# P34
# 1185020 sequences classified (23.94%)
/media/lvieira/kraken2-master/kraken2 --db /media/hd4/lucasvieira/Databases/BacteriaDB --threads 120 --gzip-compressed /media/hd4/lucasvieira/Tese/SP34-P34/Metagenome34/contigs.fasta.gz --output SP34_BacteriaNCBI_K2_Metagenome.txt --use-names --report ReportSP34_BacteriaNCBI_K2_Metagenome.txt


# Reclassification with Bracken
for i in {12..34}
do
	cd P"$i"
	/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP"$i"_BacteriaNCBI_K2_Metagenome.txt -o P"$i".F_bracken.txt -r 151 -l F -t 10
	/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP"$i"_BacteriaNCBI_K2_Metagenome.txt -o P"$i".G_bracken.txt -r 151 -l G -t 10
	/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP"$i"_BacteriaNCBI_K2_Metagenome.txt -o P"$i".S_bracken.txt -r 151 -l S -t 10
	cd ../
done

/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP10_BacteriaNCBI_K2_Metagenome.txt -o P10.F_bracken.txt -r 151 -l F -t 10
/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP10_BacteriaNCBI_K2_Metagenome.txt -o P10.G_bracken.txt -r 151 -l G -t 10
/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP10_BacteriaNCBI_K2_Metagenome.txt -o P10.S_bracken.txt -r 151 -l S -t 10

for i in {12..34}
do
	cd P"$i"
	/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP"$i"_BacteriaNCBI_K2_Metagenome.txt -o P"$i".O_bracken.txt -r 151 -l O -t 10
	cd ../
done

/media/lvieira/Bracken-master/bracken -d /media/hd4/lucasvieira/Databases/BacteriaDB -i ReportSP10_BacteriaNCBI_K2_Metagenome.txt -o P10.O_bracken.txt -r 151 -l O -t 10

# Discussion
# https://github.com/jenniferlu717/Bracken/issues/12
# https://github.com/smdabdoub/kraken-biom

# Alpha Diversity
for i in {12..34}
do
	cd P"$i"
	echo Ordem "$i" >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P"$i".O_bracken.txt -a Sh >> /media/lvieira/OTU/ShannonIndex.txt
	echo Familia "$i" >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P"$i".F_bracken.txt -a Sh >> /media/lvieira/OTU/ShannonIndex.txt
	echo Genero "$i" >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P"$i".G_bracken.txt -a Sh >> /media/lvieira/OTU/ShannonIndex.txt
	echo Especie "$i" >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P"$i".S_bracken.txt -a Sh >> /media/lvieira/OTU/ShannonIndex.txt
	cd ../
done

# Alpha Diversity Simpson
for i in {3..9}
do
	cd P0"$i"
	echo Ordem 10 >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P10.O_bracken.txt -a Si >> /media/lvieira/OTU/ShannonIndex.txt
	echo Familia 10 >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P10.F_bracken.txt -a Si >> /media/lvieira/OTU/ShannonIndex.txt
	echo Genero 10 >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P10.G_bracken.txt -a Si >> /media/lvieira/OTU/ShannonIndex.txt
	echo Especie 10 >> /media/lvieira/OTU/ShannonIndex.txt
	python /media/lvieira/KrakenTools-master/DiversityTools/alpha_diversity.py -f P10.S_bracken.txt -a Si >> /media/lvieira/OTU/ShannonIndex.txt
	cd ../
done

# Beta Diversity
../KrakenTools-master/DiversityTools/beta_diversity.py -i \
P02/P02.O_bracken.txt \
P03/P03.O_bracken.txt \
P04/P04.O_bracken.txt \
P05/P05.O_bracken.txt \
P06/P06.O_bracken.txt \
P07/P07.O_bracken.txt \
P08/P08.O_bracken.txt \
P09/P09.O_bracken.txt \
P10/P10.O_bracken.txt \
P12/P12.O_bracken.txt \
P13/P13.O_bracken.txt \
P14/P14.O_bracken.txt \
P15/P15.O_bracken.txt \
P16/P16.O_bracken.txt \
P17/P17.O_bracken.txt \
P18/P18.O_bracken.txt \
P19/P19.O_bracken.txt \
P20/P20.O_bracken.txt \
P21/P21.O_bracken.txt \
P22/P22.O_bracken.txt \
P23/P23.O_bracken.txt \
P24/P24.O_bracken.txt \
P25/P25.O_bracken.txt \
P26/P26.O_bracken.txt \
P27/P27.O_bracken.txt \
P28/P28.O_bracken.txt \
P29/P29.O_bracken.txt \
P30/P30.O_bracken.txt \
P31/P31.O_bracken.txt \
P32/P32.O_bracken.txt \
P33/P33.O_bracken.txt \
P34/P34.O_bracken.txt \
--type bracken >> BetaDiversityO.txt

# Combining files (using bracken results to continue the analysis line)
../KrakenTools-master/combine_kreports.py -r \
P02/ReportSP2_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P03/ReportSP03_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P04/ReportSP04_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P05/ReportSP05_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P06/ReportSP06_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P07/ReportSP07_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P08/ReportSP08_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P09/ReportSP09_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P10/ReportSP10_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P12/ReportSP12_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P13/ReportSP13_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P14/ReportSP14_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P15/ReportSP15_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P16/ReportSP16_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P17/ReportSP17_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P18/ReportSP18_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P19/ReportSP19_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P20/ReportSP20_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P21/ReportSP21_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P22/ReportSP22_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P23/ReportSP23_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P24/ReportSP24_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P25/ReportSP25_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P26/ReportSP26_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P27/ReportSP27_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P28/ReportSP28_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P29/ReportSP29_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P30/ReportSP30_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P31/ReportSP31_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P32/ReportSP32_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P33/ReportSP33_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
P34/ReportSP34_BacteriaNCBI_K2_Metagenome_bracken_species.txt \
-o combined.kreport

# Converting Files
for i in {12..34}
do
	cd P"$i"
	/media/lvieira/KrakenTools-master/kreport2mpa.py -r ReportSP"$i"_BacteriaNCBI_K2_Metagenome_bracken_species.txt -o BrackenSpecies"$i".mpa.txt
	cd ../
done

# Combining MPA files
/media/lvieira/KrakenTools-master/combine_mpa.py -i \
P02/BrackenSpecies02.mpa.txt \
P03/BrackenSpecies03.mpa.txt \
P04/BrackenSpecies04.mpa.txt \
P05/BrackenSpecies05.mpa.txt \
P06/BrackenSpecies06.mpa.txt \
P07/BrackenSpecies07.mpa.txt \
P08/BrackenSpecies08.mpa.txt \
P09/BrackenSpecies09.mpa.txt \
P10/BrackenSpecies10.mpa.txt \
P12/BrackenSpecies12.mpa.txt \
P13/BrackenSpecies13.mpa.txt \
P14/BrackenSpecies14.mpa.txt \
P15/BrackenSpecies15.mpa.txt \
P16/BrackenSpecies16.mpa.txt \
P17/BrackenSpecies17.mpa.txt \
P18/BrackenSpecies18.mpa.txt \
P19/BrackenSpecies19.mpa.txt \
P20/BrackenSpecies20.mpa.txt \
P21/BrackenSpecies21.mpa.txt \
P22/BrackenSpecies22.mpa.txt \
P23/BrackenSpecies23.mpa.txt \
P24/BrackenSpecies24.mpa.txt \
P25/BrackenSpecies25.mpa.txt \
P26/BrackenSpecies26.mpa.txt \
P27/BrackenSpecies27.mpa.txt \
P28/BrackenSpecies28.mpa.txt \
P29/BrackenSpecies29.mpa.txt \
P30/BrackenSpecies30.mpa.txt \
P31/BrackenSpecies31.mpa.txt \
P32/BrackenSpecies32.mpa.txt \
P33/BrackenSpecies33.mpa.txt \
P34/BrackenSpecies34.mpa.txt \
-o Bracken_CombinedSpecies.mpa.txt

#------------# 
# Krona plot #
#------------#

python3 ../../KrakenTools-master/kreport2krona.py -r ReportSP13_BacteriaNCBI_K2_Metagenome.txt -o P13.krona
ktImportText P13.krona -o P13.krona.html

python3 ../KrakenTools-master/kreport2krona.py -r AllSitesKreport.txt -o AllSitesKreport.krona
ktImportText AllSitesKreport.krona -o SupplementarFile3.html

# Results
for i in {12..34}
do
	cd P"$i"
	echo "$i"
	cat ReportSP"$i"_BacteriaNCBI_K2_Metagenome.txt | cut -f 4 | grep "S" | wc -l
	cat ReportSP"$i"_BacteriaNCBI_K2_Metagenome_bracken_species.txt | cut -f 4 | grep "S" | wc -l
	cd ../
done